### ISSUE TITLE
BUG - Sometimes Lands button doesn't work in Deck Editor for a draft tournament.

### ISSUE THREAD

**Issue by tiera3** [magefree/mage#12731]
This has happened to a different player on each of three different days.

First time -  new player, didn't get a solution in time.
Second time -  new client solved it
Third time (me) - two new clients solved it

**Comment by JayDi85** [comment]
Are you talking about disabled/hidden land button or that it does nothing on press? Lands button allowed for some game modes only. Here is some details: https://github.com/magefree/mage/issues/11877#issuecomment-2159233892

**Comment by tiera3** [comment]
I am talking about during initial deck construction (after drafting and before the first game).

**Comment by tiera3** [comment]
I cannot talk for the other people, but when it happened to me, I was reusing a client window that had previously disconnected from the server, and I reconnected with it. (I don't know if that is relevant.)

I had this happen to me a second time, so a fourth event that I know of. And this time a new client didn't fix it and I didn't get a second new client in time. I tried going ahead with round 1 of the tournament (with the default lands it assigned planning to adjust during sideboarding) and the lands button also failed to work during sideboarding. After that I connected with a third client and went to single matches for r2 & r3 and was able to adjust lands normally.

**Comment by JayDi85** [comment]
> that had previously disconnected from the server, and I reconnected with it

yes, reconnection can be a reason — e.g. miss some table events with important data, or app used old/outdated deck editor dialog after reconnect. I’ll check it.
